File: result/translate/transcoder/java2python/python/MINIMUM_NUMBER_OF_JUMPS_TO_REACH_END_OF_A_GIVEN_ARRAY_1.py
def f_gold ( arr , n ) :
    jumps = [ ]
    i , j = 0 , 0
    if n == 0 or arr [ 0 ] == 0 :
        return int ( 'inf' )
    jumps.append ( 0 )
    for i in range ( 1 , n ) :
        jumps.append ( int ( 'inf' ) )
        for j in range ( i ) :
            if i <= j + arr [ j ] and jumps [ j ] != int ( 'inf' ) :
                jumps [ i ] = min ( jumps [ i ] , jumps [ j ] + 1 )
                break
    return jumps [ n - 1 ]

File: dataset/java2cpp2(BatFix)/java/MINIMUM_NUMBER_OF_JUMPS_TO_REACH_END_OF_A_GIVEN_ARRAY_1.java
private  int f_gold ( int [ ] arr, int n ) {
  int jumps [ ] = new int [ n ];
  int i, j;
  if ( n == 0 || arr [ 0 ] == 0 ) return Integer . MAX_VALUE;
  jumps [ 0 ] = 0;
  for ( i = 1;
  i < n;
  i ++ ) {
    jumps [ i ] = Integer . MAX_VALUE;
    for ( j = 0;
    j < i;
    j ++ ) {
      if ( i <= j + arr [ j ] && jumps [ j ] != Integer . MAX_VALUE ) {
        jumps [ i ] = Math . min ( jumps [ i ], jumps [ j ] + 1 );
        break;
      }
    }
  }
  return jumps [ n - 1 ];
}